daemon/snapshotter/mount.go

```diff
@@ -145,3 +138,7 @@ func (m mounter) Mount(mounts []mount.Mount, containerID string) (string, error)
 func (m mounter) Unmount(target string) error {
 	return unmount(target)
 }
+
+func (m mounter) target(containerID string) string {
+	return filepath.Join(m.home, mountsDir, m.snapshotter, containerID)
```

Comment:
Looks like this is now the only location where `mountsDir` is used; we could either move the const close this, or just inline `"rootfs"`

Reply:
Can update quickly

Reply:
Done.

Reply:
saw it, thanks!